bands/site_sections/band_admin.py

```diff
@@ -49,40 +52,40 @@ def add_band(self, session, message='', **params):
         }
 
     @ajax
-    def mark_as_band(self, session, group_id):
+    def mark_as_band(self, session, group_id, group_type):
```

Comment:
We definitely need to check to make sure `group_type` is not an empty string. If you click "Mark as Group" without any group type selected it throws the following error: `ValueError: '' not a valid option out of {135404676: 'Guest', 147019101: 'Band'}`

Which actually would be _okay_ under normal circumstances. However, it completely deadlocks my development instance every time I do that. Like... I have to restart the server to get _any_ pages to respond. Could be related to `BADGE_LOCK`? That's the only thing I know of that would deadlock the server.

In any case, we gotta check for valid values 😄

Reply:
Oh JEEZ, big oversight! Will add that check in!

Super disconcerting that it could lock up the server though... I thought we had that totally fixed. :(